asv/commands/publish.py

```diff
@@ -83,6 +83,18 @@ def check_benchmark_params(name, benchmark):
             raise ValueError(msg)
 
 
+def safe_branch_name(branch):
+    """
+    Convert a branch name to a string, dealing with the None default value
+    """
+    if branch is None:
+        # Occurs only if no branch is set in the configuration, and is
+        # not visible to the user.
+        return "master"
```

Comment:
yes, it only affects the directory name. it''s not visible to the user or passed back to the repository